src/main/steps/applicant/keep-details-private/start_alternative/content.ts

```diff
@@ -95,15 +95,10 @@ export const form: FormContent = {
                   label: l => l.address,
                   value: 'address',
                 },
-                {
-                  name: 'contactDetailsPrivate',
-                  label: l => l.postalAddress,
-                  value: 'postalAddress',
-                },
                 {
                   name: 'contactDetailsPrivate',
                   label: l => l.Phone_number,
-                  value: 'phone',
+                  value: 'phoneNumber',
```

Comment:
@hmcts-abduldaim We can't change this as backend enum has this value, we need to change in C100 journey I think. Let's discuss